live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -266,3 +266,45 @@
       ## Record command exit failures (execve result is not command result)
       -a exit,always -F a0!=0 -S exit_group
       -a exit,always -F a0!=0 -S exit
+
+#
+# By default, the ulimit for core files is set to 0, and the default
+# filename and location for a core file is 'core' in the cwd. Update
+# limits.conf to allow processes running as root or a regular user to
+# make core files.
+#
+- lineinfile:
```

Comment:
this could be simplified:
```
+- lineinfile:
+    create: yes
+    dest: /etc/security/limits.conf
+    line: "{{ item }} soft core unlimited"
+  with_items:
+    - root
+    - *
```
but if you prefer to use `domain`, that's fine with me.